benches/lua.rs

```diff
@@ -9,167 +8,121 @@ use vector::{
     Event,
 };
 
-fn add_fields(c: &mut Criterion) {
+fn bench_add_fields(c: &mut Criterion) {
+    let event = Event::new_empty_log();
+
     let key = "the key";
     let value = "this is the value";
 
-    let value_bytes_native = Bytes::from(value).into();
-    let value_bytes_v1 = Bytes::from(value).into();
-    let value_bytes_v2 = Bytes::from(value).into();
+    let mut group = c.benchmark_group("lua_add_fields");
+    group.throughput(Throughput::Elements(1));
 
-    c.bench(
```

Comment:
Switched away from `c.bench` to creating a benchmark group given `c.bench` is deprecated: https://docs.rs/criterion/0.3.3/src/criterion/lib.rs.html#1634